### ISSUE TITLE
error when running run_scenicplus

### ISSUE THREAD

**Issue by longfeili5170** [aertslab/scenicplus#43]
Hi, Authors.Thank you for developing such a great tool.
When I run run_scenicplus for my multiome data, I get an error like the following:
<img width="940" alt="屏幕快照 2022-09-29 下午3 55 30" src="https://user-images.githubusercontent.com/68364890/192975508-360bfa58-b2e0-42ae-bfa3-404d86b94a6d.png">
This error seems to have appeared in previous versions.
Any suggestion to solve this problem？
Thanks！

**Comment by SeppeDeWinter** [comment]
Hi @longfeili5170 

Calculating TF-to-gene links failed for some reason. 

Could you show the complete output of the command. The command should print some additional information about the exception that occurred but it is not visible in your screenshot.

Best,

Seppe

**Comment by longfeili5170** [comment]
The reason seems to be a problem with an array.
<img width="788" alt="屏幕快照 2022-09-29 下午5 12 28" src="https://user-images.githubusercontent.com/68364890/192992286-0b759718-a216-48cb-a4a9-8d002d466b4b.png">

<img width="779" alt="屏幕快照 2022-09-29 下午5 09 11" src="https://user-images.githubusercontent.com/68364890/192992334-c0e1761d-70df-44da-8e79-14a7a8c1fe72.png">

**Comment by longfeili5170** [comment]
Hi @SeppeDeWinter 

I still can't resolve the error.

Is there any suggestion? Looking forward to your reply.

**Comment by SeppeDeWinter** [comment]
Hi @longfeili5170 

Sorry I lost track of this issue, so thanks for poking me!

Could you please copy-paste the entire output (not just screenshots), it's very difficult to troubleshoot what is going wrong from these two screenshots.

Could you also show the output of 

```python

scplus_obj.X_EXP

```

and 


```python

scplus_obj.to_df('EXP')

```

Thanks

**Comment by longfeili5170** [comment]
Thank you for your reply. @SeppeDeWinter 

Considering the output is too long, I put the jupyter here:
http://139.196.189.180:8081/scenicplus/scenicplus-error.html

Thanks!

**Comment by SeppeDeWinter** [comment]
Thanks for the extensive output.

I'm still not entirely sure what is going wrong...

You might be running out of memory and/or disk space, see: 

```
file_system_monitor.cc:105: /home/lilongfei/tmp/ray_spill/session_2022-09-29_10-35-42_640665_108198 is over 95% full, available space: 2085549563904; capacity: 53953699999744. Object creation will fail if spilling is required.
```

Could you try setting `_temp_dir ` to a location which has more space available? (not sure wether this is the problem)


Also there seems to be a lot of zeros in your expression matrix (maybe there are some cells with zero counts?)

Could you show me the output of:

```python

scplus_obj.to_df('EXP').sum(1).sort_values()

```

and

```python

scplus_obj.to_df('EXP').sum(0).sort_values()

```

and 

```python

len(np.unique(scplus_obj.X_EXP))

```

Thanks!

Best,

Seppe

**Comment by longfeili5170** [comment]
Hi !

I used a server with bigger memory to run and it worked !

Thanks!
